### ISSUE TITLE
leverage k-means clustering in value-similarity.py

### ISSUE THREAD

**Issue by harsham05** [chrismattmann/tika-similarity#41]
Thank you @thammegowda for the valuable suggestion.

k-means will iteratively cluster the data based on the vicinity of metadata values.

**Comment by chrismattmann** [comment]
@harsham05 you have a patch / pull here right? Can you send?

**Comment by harsham05** [comment]
@chrismattmann k-means is in my fork. But the current implementation is incredibly slow. I will send a PR after refactoring my code to take into account number of documents to cluster (for calculating the number of iterations)
https://github.com/harsham05/tika-similarity

Usage: Running below directly generates clusters.json
```
python k-means.py --inputDir ~/Documents/CS572DR/dataset__images/polr_jpg/
```

**Comment by chrismattmann** [comment]
@harsham05 please send what you have now. We can update the PR later, but I'd like to see it now.

**Comment by harsham05** [comment]
@chrismattmann Please find the same here
https://github.com/chrismattmann/tika-similarity/pull/46

**Comment by chrismattmann** [comment]
fixed in #46 thanks @harsham05
